pages/product.html

```diff
@@ -126,9 +126,7 @@
                     <div class="card-content">
                         <h2 class="card-heading">Black Zari Banarasi Saree with Blouse
                         </h2>
-                        <p class="card-author">Rs. 3,390 only</p
-                    </div>
-                    <div class="card-actions">
+                        <p class="card-author">Rs. 3,390 only</p </div> <div class="card-actions">
```

Comment:
&gt; missing in closing &lt;/p&gt; tag.  Also reformat div.card-actions to next line.